setup.py

```diff
@@ -5,9 +5,9 @@
     long_description = readme.read()
 
 setup(
-    name='edx-ecommerce-api-client',
+    name='edx-rest-api-client',
     version='1.1.2',
-    description='Slumber client used to access APIs exposed by the edX E-Commerce Service',
+    description='Slumber client used to access various edX Platform REST APIs.',
```

Comment:
Is it important to call-out that Slumber is being utilized in the description of this package?